README.md

```diff
@@ -7,9 +7,10 @@ code collaboration protocol and stack.
 
 **🚨 WORK IN PROGRESS 🚨**
```

Comment:
Should we remove this now since Radicle Link is ~* live *~ ?

Reply:
I'll remove it when we fully release 😝